### ISSUE TITLE
Access sdcard

### ISSUE THREAD

**Issue by shapeshifter910** [kiwix/kiwix-android#2596]
Should the app not ask for permission to access the sdcard?
Feels like on my tablet (Samsung Galaxy Tab S) with Android 6.1 Kiwix finds the Wikipedia zim file but never loads any content from the Wikipedia.
So it's not working.

**Comment by shapeshifter910** [comment]
A small update. I've downloaded a few small wikis and tried to open them. They opened while I was online.
Once I switched off my wifi, Kiwix crashed when trying to open the same wikis again.
And what's up with that always spinning circle as soon as I open Kiwix?
At this point I have to conclude the Android app on, at least Samsung Galaxy tablets, is not functional at all.
And it looks like it's "pre-alpha" quality coding in general.
Too bad as I really need something like this for my offgrid place.
